features/support/hooks.el

```diff
@@ -0,0 +1,12 @@
+(require 'ecukes)
+
+(defun kill-js-buffers ()
+  (mapcar (lambda (buffer)
+            (when (string-match "\.js" (buffer-name buffer))
```

Comment:
Yes, you can use `or` like that.

Thinking about it, it's probably better to try and restore the state as much as possible to avoid polluting the test environment for the next test. I'm thinking something like this:

```lisp
(defvar mocha-init-buffers (buffer-list))

(defun mocha-kill-buffers ()
  (mapc
   (lambda (buffer)
     (unless (member buffer mocha-init-buffers)
       (kill-buffer buffer)))
   (buffer-list)))

(Before
 (mocha-kill-buffers))
```